src/stateManagement/stackSpecificStateManager.js

```diff
@@ -96,6 +97,7 @@ function addStackStateManager(element, otherTools) {
     'referenceLines',
     'crosshairs',
     'stackRenderer',
+    BaseBrushTool.getReferencedToolDataName(),
```

Comment:
So all tools that extend `BaseBrush` will use this name for state lookups? Are we using this the way it's intended, or is this worth rethinking?

Reply:
Let me check... I think we can nuke this now we aren't storing each `Labelmap2D` as an annotation on the tool manager.

Reply:
Nuked.